user/python-ssh2/template.py

```diff
@@ -0,0 +1,30 @@
+pkgname = "python-ssh2"
+pkgver = "1.0.0"
+pkgrel = 0
+build_style = "python_pep517"
+make_build_env = {
+    "SSH2_PYTHON_VERSION": pkgver,
+    "SYSTEM_LIBSSH2": "1",
+}
+hostmakedepends = [
+    "python-build",
+    "python-cython",
+    "python-installer",
+    "python-setuptools",
+]
+makedepends = [
+    "libssh2-devel",
+    "python-devel",
+]
+checkdepends = [
+    "python-jinja2",
+    "python-pytest",
+]
+pkgdesc = "Python bindings for libssh2"
+maintainer = "Erica Z <zerica@callcc.eu>"
+license = "LGPL-2.1-only"
+url = "https://github.com/ParallelSSH/ssh2-python"
+source = f"{url}/archive/refs/tags/{pkgver}.tar.gz"
+sha256 = "70c6b6efd8ca9f8de9c2d77e7cb1d5859542588347ea426d6822b0ffd9889af3"
+# check: ModuleNotFoundError: No module named 'ssh2.session'
```

Comment:
this is probably worth looking into first

Reply:
if there's no tests on pypi then that is a reason to not use it

Reply:
as for the error, it's probably the usual compiled module cwd thing